packages/prism-react-renderer/src/utils/__tests__/normalizeTokens.test.ts

```diff
@@ -26,7 +27,7 @@ describe("normalizeTokens", () => {
         type: "test2",
         content: "world",
       },
-    ]
+    ] as Token[]
```

Comment:
TypeScript is a bit more strict about mock data structures, so we need to coerce these.